src/components/admin/article/components/TitleField.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { RequiredTextField } from 'components/admin/form-components/validated-fields';
+
+const TitleField = (props) => {
+  const onChange = (event) => {
```

Comment:
As i was refactoring stuff for the linter I double checked this as I tried making one of your components a functional one to make the linter not complain there were 2 components in a file, and this has the same problem as binding event handlers. So you probably should make this a component so that it can have a this handler even though this looks much nicer imo